pkg/apmintegration/helpers.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package apmintegration
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"strings"
+)
+
+func getGithubOrg(url string) (string, error) {
```

Comment:
Are we only supporting `github` repos?

Reply:
apm itself doesn't only support github repos, apm can support an arbitrary git server but has a concept of organization/repository-name (which sounds github-ish), but these are only used for its internal aliasing to short-hand/make human readable a long url (e.g org/repo (alias)  => https://www.github.com/org/repo.git (url))

Reply:
renamed, we should support any repo but it wouldn't totally surprise me if we hit a bug for a user using gitlab or something